extensions/amp-story/1.0/amp-story.js

```diff
@@ -1444,7 +1447,9 @@ export class AmpStory extends AMP.BaseElement {
         actions.execute(this.sidebar_, 'open', /* args */ null,
         /* source */ null, /* caller */ null, /* event */ null,
             ActionTrust.HIGH);
+        this.openMask_()
```

Comment:
Nit: since this file is unfortunately a 2k+ lines monster with a lot of different things, what do you think of adding something like `opacity` in the same, ie `openOpacityMask`?

Reply:
Makes perfect sense to me!